### PR TITLE
add banner to account page

### PR DESCRIPTION
**Pull request by DilwoarH**
PRE REQUISITE:  https://github.com/alphagov/digitalmarketplace-user-frontend/pull/43

Add the user research banner to the account page using the existing GOV.uk pattern. Allow for links to 'sign up to mailing list' page

As a user I should be presented with the banner to sign up to the user research mailing list when I log in if: I have not already closed it and I have not already signed up for this.
This is the first point of contact for getting users to sign up to the mailing list.

The close link should set a cookie value of 'seen_this_banner_thing' to true so we don't show it again
The sign up link should link to the new sign up page.

https://trello.com/c/OJ9lFk1M/129-add-banner-to-account-page

screen shot 2018-03-13 at 14 42 48

### CODE REVIEW

**Comment by MuriloDalRi** [comment]
Approved too soon... Looks good except for travis

File: app/templates/buyers/index.html [hunk 1]
```diff
@@ -24,28 +30,42 @@
...
+  </div>
+  <div class="column-one-third dmspeak">
+    <h2 class="heading-xmedium">Account settings</h2>
     <p>
-      {% if user_briefs_total %}
-        <a href="{{ url_for('buyers.buyer_dos_requirements') }}">View your requirements</a><br>  
-      {% else %}   
-        You don't have any requirements
-      {% endif %}
+      <a href="{{ url_for('external.user_research_consent') }}">
```
Comment (benvand) [inline app/templates/buyers/index.html]: We're happy with this below the other notifications right?

**Comment by DilwoarH** [comment]
@benvand - updated with tests

File: tests/main/views/test_notifications.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    ):
+        with self.app.test_client():
+            self.login_as_buyer(user_researh_opted_in=False)
+            res = self.client.get('/buyers')
+            assert res.status_code == 200
+            assert 'Help us improve the Digital Marketplace' in res.get_data(as_text=True)
+            assert 'Sign up to be a potential user research participant' in res.get_data(as_text=True)
+            assert '<a href="#" class="user-research-banner-close-btn">Close</a>' in res.get_data(as_text=True)
+            cookie_value = self.get_cookie_by_name(res, 'seen_user_research_message')
+            assert cookie_value is None
```
Comment (DilwoarH) [inline tests/main/views/test_notifications.py]: @benvand - added check to make sure cookie value isn't set

File: app/templates/buyers/index.html (hunk 1 above)
Comment (benvand) [reply app/templates/buyers/index.html]: Looked on local, it's in the righ place
 👍

**Pull request merged**